packages/charts/src/vaadin-chart.js

```diff
@@ -549,12 +549,10 @@ class Chart extends ResizeMixin(ElementMixin(ThemableMixin(PolymerElement))) {
     deepMerge(options, this.additionalOptions);
 
     if (this.type) {
-      options.chart ||= {};
```

Comment:
Removed this line as not needed here and below, as it's present in the base config:

https://github.com/vaadin/web-components/blob/ed28a0f1f3b2a40ed26c25ffbfe915a1487b9cc3/packages/charts/src/vaadin-chart.js#L490